imagebuildah/executor.go

```diff
@@ -350,8 +363,44 @@ func (b *Executor) waitForStage(ctx context.Context, name string, stages imagebu
 func (b *Executor) getImageTypeAndHistoryAndDiffIDs(ctx context.Context, imageID string) (string, []v1.History, []digest.Digest, error) {
 	b.imageInfoLock.Lock()
 	imageInfo, ok := b.imageInfoCache[imageID]
+
+	// Wrap unexported data in cache structures for JSON Logger
+	exporter := func(wrapper string, data interface{}) (string, envelopExecutorJSON) {
```

Comment:
Instead of re-exporting unexported structures like that, it could be "fixed" in place as well. But that would impact many other places that use the established names.